trunk/PO-9_210661/task_03/src/CMakeLists.txt

```
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 3.5)
+project(task_03 VERSION 2023.10.04.0 LANGUAGES CXX)
```

```suggestion
project(task_03   LANGUAGES CXX)
```
Это ж дата, либо убирать либо менять)